### PR TITLE
test: added E2E coverage for `isAuthorizedRaw()` in HIP-632.

### LINKED ISSUE #844: feat: examples/tests for `isAuthorizedRaw` (of HIP-632)
### Problem
The `isAuthorized()` function from HIP-632 has been fully implemented in the services. This feature requires test coverage to ensure its proper usage.

### Solution

- Write tests to cover:
    - Verify  isAuthorizedRaw correctly returns true when the signatureBlob has been validated to have been signed by the evm key of a given address.
    - Verify  isAuthorizedRaw correctly returns false when the signatureBlob does not validate to have been signed by the evm key of a given address.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**:
This PR adds e2e test coverage for isAuthorizedRaw() in HIP-632

**Related issue(s)**:

Fixes #844

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test/system-contracts/hedera-account-service/crypto-allowance/cryptoAllowance.js [hunk 1]
```diff
@@ -302,4 +308,95 @@ describe('@CryptoAllowance Test Suite', () => {
...
+    const correctSignerReceiptResponse = correctSignerReceipt.logs.find(
+      (l) => l.fragment.name === 'IsAuthorizedRaw'
+    ).args;
+
+    expect(correctSignerReceiptResponseCode).to.eq(22n);
+    expect(correctSignerReceiptResponse[0].toLowerCase()).to.eq(
+      signerAlias.toLowerCase()
+    );
+    expect(correctSignerReceiptResponse[1]).to.be.true;
+  });
```
Comment (lukelee-sl) [inline test/system-contracts/hedera-account-service/crypto-allowance/cryptoAllowance.js:401]: Maybe there could be one or two negative case tests?

Comment (quiet-node) [reply test/system-contracts/hedera-account-service/crypto-allowance/cryptoAllowance.js:401]: Ah yeah I grouped the negative case within the one that used ECDSA key https://github.com/hashgraph/hedera-smart-contracts/pull/997/files#diff-90094514c9235785c54135c79d4a4ea158720b3ac9c2ba4ca0a175b5aa0ffe8eR338-R357. 

But yeah let me separate it out and add another case for ED key.

Comment (quiet-node) [reply test/system-contracts/hedera-account-service/crypto-allowance/cryptoAllowance.js:447]: Updated

**Review by lukelee-sl — APPROVED** [review]
LGTM

**Pull request merged**